Tests/KituraSampleRouterTests/KituraTest.swift

```diff
@@ -41,18 +41,22 @@ extension KituraTest {
 
     func performServerTest(asyncTasks: @escaping (XCTestExpectation) -> Void...) {
         let router = RouterCreator.create()
-        Kitura.addHTTPServer(onPort: 8090, with: router)
-        Kitura.start()
-        sleep(1)
```

Comment:
Yes, they're not needed there either. There and in your changes, it does not hurt anything really, just added complexity that is not required. Examples where the lifecycle methods are required are in [KituraNetTests.LifecycleListenerTests](https://github.com/IBM-Swift/Kitura-net/blob/master/Tests/KituraNetTests/LifecycleListenerTests.swift)

The sleep() calls used to be required earlier because listenSocket.listen() used to be invoked [asynchronously](https://github.com/IBM-Swift/Kitura-net/commit/0e81e12d2af3befa23e31e5690616c540a40d8eb#diff-87121335409889e3d4ed677d613a1612L131), and now it is invoked [synchronously](https://github.com/IBM-Swift/Kitura-net/commit/0e81e12d2af3befa23e31e5690616c540a40d8eb#diff-87121335409889e3d4ed677d613a1612R67)